packages/caravan-fees/src/types.ts

```diff
@@ -0,0 +1,76 @@
+import BigNumber from "bignumber.js";
+import { Transaction } from "bitcoinjs-lib-v5";
```

Comment:
Noted out of band, but let's make sure the dependency is declared in package.json and we use the latest version here.

Reply:
this needs to be done

Reply:
[done](https://github.com/caravan-bitcoin/caravan/pull/114/files#diff-a3e31745046775327f41608ad7eccffc05717d319e1f0fb67360040d4488fd61)